deploy/docker/docker-entrypoint.sh

```diff
@@ -153,14 +153,36 @@ do
     fi
 done
 
-## EMQ Plugin load settings
+## EMQX Plugin load settings
 # Plugins loaded by default
 
 if [[ ! -z "$EMQX_LOADED_PLUGINS" ]]; then
     echo "EMQX_LOADED_PLUGINS=$EMQX_LOADED_PLUGINS"
     # First, remove special char at header
     # Next, replace special char to ".\n" to fit emq loaded_plugins format
-    echo $(echo "$EMQX_LOADED_PLUGINS."|sed -e "s/^[^A-Za-z0-9_]\{1,\}//g"|sed -e "s/[^A-Za-z0-9_]\{1,\}/\. /g")|tr ' ' '\n' > /opt/emqx/data/loaded_plugins
+    for var in $(echo $(echo "$EMQX_LOADED_PLUGINS."|sed -e "s/^[^A-Za-z0-9_]\{1,\}//g"|sed -e "s/[^A-Za-z0-9_]\{1,\}/\ /g")|tr ' ' '\n'); do
+        if [ ! -z $(grep -o $var ${_EMQX_HOME}/data/loaded_plugins) ]; then
+            echo "$(sed -r "s/\{($var), (true|false)\}./\{\1, true\}./1" ${_EMQX_HOME}/data/loaded_plugins)" > ${_EMQX_HOME}/data/loaded_plugins
```

Comment:
to allow any number of white spaces after `,`
```suggestion
            echo "$(sed -r "s/\{($var),[]*(true|false)\}./\{\1, true\}./1" ${_EMQX_HOME}/data/loaded_plugins)" > ${_EMQX_HOME}/data/loaded_plugins
```